regen_openapi.sh

```diff
@@ -10,4 +10,6 @@ yarn openapi-generator-cli generate -i openapi.json -g python -o python/ -c pyth
 
 yarn openapi-generator-cli generate -i openapi.json -g typescript -o javascript/src/openapi -c javascript/openapi-generator-config.json
 
-yarn openapi-generator-cli generate -i openapi.json -g go -o go/internal/openapi -c go/openapi-generator-config.json
\ No newline at end of file
+yarn openapi-generator-cli generate -i openapi.json -g go -o go/internal/openapi -c go/openapi-generator-config.json
+
+yarn openapi-generator-cli generate -i openapi.json -g java -o java -c java/openapi-generator-config.json
```

Comment:
Is it possible to generate the openapi code into it's own subfolder like `java/openapi` and then gitignore it?

Reply:
Done.

Reply:
I'm still seeing the code generated to java/ i think that force push may have removed some of your changes

Reply:
I changed it in the openapi-generator-config.json (the package). 
Do you mean that we should have all generated files (including the non-Java code) in an internal file?

Reply:
Why are the generated files even committed to the repo? They shouldn't be... Add the directory to .gitignore, and remove all of the files...

Reply:
ah got it
I think I can just add all the bin to the gitignore, another option is just to create everything inside an inner folder. I see that this is what you did for golang. I can do either

Reply:
Inner folder would probably be the the best, and then as @tasn mentioned add that folder to the `java/.gitignore` file.

Reply:
sg, done

Reply:
looks like the most recent changes broke some things, the internal code is now being generated to `java/internal/src/main/java/openapi...` but your bindings still expect them in `java/src/main/java/openapi`

Reply:
hmm that's weird, I changed build.gradle to internal/openapi/src/main/java and it works here. are you sure you are synced to the latest?